src/components/app/details/main.tsx

```diff
@@ -162,11 +173,9 @@ export function AppHeader({ appName }: { appName: string }) {
                         to={`${match.url}/${URLS.APP_DETAILS}`}
                         className="tab-list__tab-link"
                         onClick={(event) => {
-                            ReactGA.event({
-                                category: 'App',
-                                action: 'App Details Clicked',
-                            })
-                        }}
+                            trackEvent('App', 'App Details Clicked');
```

Comment:
Can we remove this inline function by extracting out  these two in a common function